### PR TITLE
add generic-amd64-fs yocto-machine

### PR DESCRIPTION
**Pull request by rcooke-warwick**
This is to resolve build and deploy pipeline issues - perviously we were breaking our pipeline conventions which created a cascade of errors

Change-type: patch

### CODE REVIEW

File: layers/meta-balena-generic/conf/machine/generic-amd64-fs.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#@TYPE: Machine
+#@NAME: Generic amd64 fs
+#@DESCRIPTION: Machine configuration for Generic amd64 fs
+
+MACHINEOVERRIDES = "generic-amd64-fs:${MACHINE}"
+include conf/machine/generic-amd64.conf
```
Comment (alexgg) [inline layers/meta-balena-generic/conf/machine/generic-amd64-fs.conf:6]: This is probably OK as the assignment is not immediate - however check that the MACHINEOVERRIDES actually contains all the required machines. If not, move the include before the assignment.

**Review by alexgg — CHANGES_REQUESTED** [review]
lgtm, just one small check

File: layers/meta-balena-generic/conf/machine/generic-amd64-fs.conf (hunk 1 above)
Comment (rcooke-warwick) [reply layers/meta-balena-generic/conf/machine/generic-amd64-fs.conf:6]: @alexgg in this case the machine name is purely cosmetic - it is just a rename of `generic-amd64` - in general is it just better to put the include before hand just to be sure?

Comment (alexgg) [reply layers/meta-balena-generic/conf/machine/generic-amd64-fs.conf]: @rcooke-warwick it is fine - I checked and the MACHINEOVERRIDES are the same as `generic-amd64`:
```
MACHINE=generic-amd64 bitbake balena-image-flasher -e | grep ^MACHINEOVERRIDES=
MACHINEOVERRIDES="generic-amd64"
```
And:
```
MACHINE=generic-amd64-fs bitbake balena-image-flasher -e | grep ^MACHINEOVERRIDES=
MACHINEOVERRIDES="generic-amd64:generic-amd64-fs"
```

**Comment by rcooke-warwick** [comment]
@resin-jenkins retest this please

**Comment by rcooke-warwick** [comment]
above retest is due to now having to remove our jenkins workaround for the mismatched machine name + device type ^

**Pull request merged**